tools/create_ontology_definition_lookup_table.py

```diff
@@ -0,0 +1,89 @@
+"""
+Gathers metadata of ontologies found in the OLS api and adds any ontologies 
+necessary that are not included in ols.
+Jira ticket FD-1381
+"""
+
+import requests
+import pandas as pd
+from datetime import date
+import sys
+import os
+
+# Specify the path to resources
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+
+from resources import ONTOLOGY_API_LOOKUP_TABLE_PATH, OLS_API_URL, MONARCH_API_URL, LOINC_API_URL
+
+JIRA_ISSUES = ['fd1381']
+export_date = date.today()
+
+# Define the base API endpoints
+ols_base_url = OLS_API_URL
```

Comment:
I'm curious about why you are assigning a local variable to a value imported from the resources? Not saying it is wrong, just curious as to the reasoning for this approach

Reply:
I like having the resources file in case we need to reuse the variables, and I will often prefer a different name than the one given in the resource file.  I might change the name to give the code more context. 

I am going to rework the names a little bit actually. To be a bit more accurate and reusable.